packages/contracts/test/L1/DepositFeed.spec.ts

```diff
@@ -164,18 +209,55 @@ describe('DepositFeed', () => {
         })
       })
 
+      it('when a contract deposits a transaction with an ETH value.', async () => {
+        // Deploy a dummy contract so we can impersonate it
+        const dummy = await (await ethers.getContractFactory('Dummy')).deploy()
+        await dummy.deployed()
+        // this is not emitting an event!
```

Comment:
It does decode an awaited event below however, is this comment wrong?

Reply:
my bad, this was a debugging comment that got left in.

Reply:
Removed the old comment in 55bf74ba0b9a48d536b98bdfff8c4d23d8a10b1d